pkgs/applications/misc/sdcv/default.nix

```diff
@@ -1,7 +1,9 @@
-{ stdenv, fetchurl, pkgconfig, glib, gettext }:
+{ stdenv, fetchurl, pkgconfig, glib, gettext, fetchFromGitHub, cmake,
+readline }:
 
 stdenv.mkDerivation rec {
-  name= "sdcv-0.4.2";
+  name = "sdcv-${version}";
+  version = "0.5.2";
 
   meta = {
     homepage = http://sdcv.sourceforge.net/;
```

Comment:
This page now redirects to https://dushistov.github.io/sdcv/ so we should set `meta.homepage` to this new home.